Makefile

```diff
@@ -115,15 +115,17 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: manifests controller-gen code-generator openapi-gen helm helm-docs ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations and client-go libraries.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./api/..."
 	./hack/update-codegen.sh $(GO_CMD) $(PROJECT_DIR)/bin
-	./hack/python-sdk/gen-sdk.sh
+	# TODO: fix python-sdk generation
+	# Nevertheless, any changes to the python-sdk should first be pushed to the upstream repo.
+	# ./hack/python-sdk/gen-sdk.sh
 
 .PHONY: fmt
 fmt: ## Run go fmt against code.
 	$(GO_CMD) fmt ./...
 
 .PHONY: fmt-verify
 fmt-verify:
-	@out=`$(GO_FMT) -w -l -d $$(find . -name '*.go')`; \
+	@out=`$(GO_FMT) -w -l -d $$(find . -name '*.go' -not -path "./vendor/*")`; \
```

Comment:
Why do we need to exclude `vendor` directory?. If we have to, can't we move it to ocp Makefile.